ImperatorToCK3/Source/Imperator/Genes/Genes.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Genes.h"
+#include "Log.h"
+#include "ParserHelpers.h"
+
+
+ImperatorWorld::Genes::Genes(const std::string& thePath)
+{
+	registerKeys();
+	parseFile(thePath);
+	clearRegisteredKeywords();
+}
+
+ImperatorWorld::Genes::Genes(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+void ImperatorWorld::Genes::registerKeys()
+{
+	/*registerKeyword("morph_genes", [this](const std::string& unused, std::istream& theStream) {
+		Log(LogLevel::Debug) << "<> just a test 1";
+		loadMorphGenes(theStream);
+	});*/
+	registerKeyword("accessory_genes", [this](const std::string& unused, std::istream& theStream) {
+		Log(LogLevel::Debug) << "<> entering accessory genes";
```

Comment:
Once you're done wit the thing, remember to scour for debugs and delete/comment them out whenever no longer necessary.

Reply:
This one slipped through, I added a lot of logging 😄